### ISSUE TITLE
GitHub Enterprise support

### ISSUE THREAD

**Issue by TheJaredWilcurt** [linyows/github-wiki-search#16]
You have the github website hardcoded. It would be nice if there was an options page for users to add in custom URLs/domains for this to work on. Many companies have the enterprise version of GitHub and being able to search the wiki at work would be super useful.

If nothing else, you could just add in a regex thing to look for `http(s)://githib.*.com/*/*/wiki`

      function allowedLocation(callback) {
        if (/^https?:\/\/github\.com\/.*?\/.*?\/wiki/.test(location.href)) {
          callback();
        }
      }

**Comment by soasme** [comment]
+1

**Comment by ccampanale** [comment]
+1

**Comment by ccampanale** [comment]
In the meantime, I made [this](https://github.com/ccampanale/github-wiki-search-modified). I'm still testing it to ensure it isn't breaking anything but it seems to be working as intended until the extension can include support for GHE installations.

**Comment by ccampanale** [comment]
I agree with @TheJaredWilcurt though I think a better regex would be `/^https?:\/\/github\..*?\/.*?\/.*?\/wiki/` as this would return true for both public and enterprise GitHub installations. That being said, I believe the real limitation here is in the manifest of the Chrome extension as you have to define on which sites you want the script to run. Though I may be wrong!

**Comment by judwhite** [comment]
+1

**Comment by kleinen** [comment]
+1

**Comment by TheJaredWilcurt** [comment]
Really the correct answer is to just let people type in the root for their own github server, even if it's just `localhost:8080`.

Though, auto-guessing would be cool too.

**Comment by ccampanale** [comment]
Hrmmm we never closed this issue... lol

This is done and it's a feature now for anyone who is curious. And you can do exactly that @TheJaredWilcurt; there's an options page that lets you define for which domains you want the extension to run on.

**Comment by TheJaredWilcurt** [comment]
@ccampanale

I just downloaded the extension and tried it on both github and ghe, and neither worked. There's also no options page.

![options](https://cloud.githubusercontent.com/assets/4629794/13829563/0436d02e-eb9e-11e5-8f31-9ea191d565f1.png)

**Comment by kleinen** [comment]
I also am unable to find the options you refer to.

**Comment by ccampanale** [comment]
Guys, my bad... I got the extension mixed up; I was thinking this was another one that I contributed to that recently got enterprise support... :-/ This one unfortunately still has no support for enterprises... :( Sorry

**Comment by yuriteixeira** [comment]
That fixed the problem: https://github.com/linyows/github-wiki-search/pull/23
